hamilton/experimental/h_spark.py

```diff
@@ -149,3 +149,289 @@ def build_result(
             return df.to_pandas()
         else:
             return df
+
+
+def numpy_to_spark_type(numpy_type: Type) -> types.DataType:
```

Comment:
No utility functinos in spark to do this right?

Reply:
I looked -- didn't find anything obvious.